app/views/collections/show.html.erb

```diff
@@ -4,42 +4,27 @@
 </div>
 
 <div class="row">
-  <div class="col-md-8 col-md-offset-2 col-xs-12 box">
+  <div class="col-xs-12 box">
 
     <% @apparels.each do |apparel| %>
   
       <div class="row">
-        <div class="col-xs-12 collection-box">
-          <div class="collection-name">
-          <%= apparel.name %>
+        
+          <div class="col-xs-5 collection-name">
+            <%= apparel.name %>
           </div>
-          <div class"col-xs-12">
-            <div class="col-xs-3 collection-image">
-                <%= image_tag("apparels/#{apparel.id}.jpg") %>
-            </div>
-            <div "coll-xs-9">
-              <div class="col-xs-12 collection-price">
-              <%= apparel.currency %> <%= apparel.price %>
-              </div>
-              <div class="col-xs-12 collection-description">
-              <%= apparel.description %>
-              </div>
-              <div class="col-xs-12 collection-material">
-              <%= apparel.material %>
-              </div>
-              <div class="col-xs-12 collection-care">
-              <%= apparel.care %>
-              </div>
-              <div class="col-xs-12 collection-size">
-              <%#= apparel.size %>
-              </div>
-            </div>
-          </div>
-          
-        </div>
+
+          <div class="col-xs-5  collection-image">
+            <%= image_tag("apparels/#{apparel.id}.jpg") %>
+          </div> 
+        
+      </div>
+
+      <div class="row">
+        <%= link_to apparel.name, apparel_path, :class => "apparel-link" %>
+        <% end %>
```

Comment:
This `<% end %>` should be somewhere else. Make sure your indentations are correct as well